src/components/Widgets/ReviewStatusMetricsWidget/MetaReviewStatusMetricsWidget.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react'
+import { FormattedMessage } from 'react-intl'
+import _indexOf from 'lodash/indexOf'
+import _isEqual from 'lodash/isEqual'
+import { WidgetDataTarget, registerWidgetType }
+       from '../../../services/Widget/Widget'
+import ReviewStatusMetrics from '../../../pages/Review/Metrics/ReviewStatusMetrics'
+import WithReviewMetrics from '../../HOCs/WithReviewMetrics/WithReviewMetrics'
+import WithChallengeReviewMetrics
+      from '../../AdminPane/HOCs/WithChallengeReviewMetrics/WithChallengeReviewMetrics'
+import WithProjectReviewMetrics
+      from '../../AdminPane/HOCs/WithProjectReviewMetrics/WithProjectReviewMetrics'
+
+import QuickWidget from '../../QuickWidget/QuickWidget'
+import messages from './Messages'
+
+const descriptor = {
+  widgetKey: 'MetaReviewStatusMetricsWidget',
+  label: messages.label,
+  targets: process.env.REACT_APP_FEATURE_META_QC === 'enabled' ?
+    [WidgetDataTarget.review, WidgetDataTarget.challenge,
+     WidgetDataTarget.challenges] : [],
+  minWidth: 2,
+  defaultWidth: 4,
+  minHeight: 4,
+  defaultHeight: 6,
+  defaultConfiguration: {
+    showByPriority: false,
+  },
+}
+
+export default class MetaReviewStatusMetricsWidget extends Component {
+  ReviewStatusMetricsComponent = null
```

Comment:
Shouldn't this be in `state`? Otherwise you aren't guaranteed that your component will rerender when you change it in `componentDidUpdate`